packages/blade/src/components/Dropdown/FilterChipSelectInput.web.tsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-unnecessary-type-assertion */
+
+import React from 'react';
+import { useDropdown } from './useDropdown';
+import { dropdownComponentIds } from './dropdownComponentIds';
+import { assignWithoutSideEffects } from '~utils/assignWithoutSideEffects';
+import { BaseFilterChip } from '~components/FilterChip/BaseFilterChip';
+import { getActionListContainerRole } from '~components/ActionList/getA11yRoles';
+import type { BaseFilterChipProps } from '~components/FilterChip/types';
+import type { DataAnalyticsAttribute } from '~utils/types';
+
+type FilterChipSelectInput = Pick<
+  BaseFilterChipProps,
+  | 'onKeyDown'
+  | 'value'
+  | 'onClearButtonClick'
+  | 'label'
+  | 'testID'
+  | 'onClick'
+  | 'selectionType'
+  | 'onBlur'
+> & {
+  accessibilityLabel?: string;
+} & DataAnalyticsAttribute;
+
+const _FilterChipSelectInput = ({
+  onClick,
+  onBlur,
+  onKeyDown,
+  accessibilityLabel,
+  testID,
+  value,
+  onClearButtonClick,
+  label,
+  ...rest
+}: FilterChipSelectInput): React.ReactElement => {
+  const {
+    onTriggerClick,
+    onTriggerKeydown,
+    dropdownBaseId,
+    isOpen,
+    activeIndex,
+    hasFooterAction,
+    triggererRef,
+    selectionType,
+  } = useDropdown();
+
+  return (
+    <BaseFilterChip
+      label={label}
+      value={value}
+      onClearButtonClick={onClearButtonClick}
+      selectionType={selectionType}
+      {...rest}
+      ref={triggererRef as any}
+      accessibilityProps={{
+        label: accessibilityLabel ?? label,
+        hasPopup: getActionListContainerRole(hasFooterAction, 'DropdownButton'),
```

Comment:
```suggestion
        hasPopup: getActionListContainerRole(hasFooterAction, 'FilterChipSelectInput'),
```

do handle it inside `getActionListContainerRole` as well